plugins/modules/postgresql_info.py

```diff
@@ -230,6 +230,7 @@ def __init__(self, module, db_conn_obj):
         self.module = module
         self.db_obj = db_conn_obj
         self.cursor = db_conn_obj.connect()
+        self.default_db = module.params['db'] or module.params['login_db'] or module.params['database']
```

Comment:
```suggestion
        self.default_db = module.params['db'] or module.params['login_db'] or module.params['database']
```
doesn't it work with just `module.params['login_db']`? the others are just aliases IIRC

Reply:
Im not sure. It seems to work - I put it since just above we override all three values separately, and the conn_params method treat them separately.

Reply:
ach, i see, i think it's something specific to reconnect(), let's try w/o it here but keeping an eye after your fix gets released, thanks!